staging/src/k8s.io/client-go/tools/clientcmd/validation.go

```diff
@@ -207,35 +207,39 @@ func validateAuthInfo(authInfoName string, authInfo clientcmdapi.AuthInfo) []err
 		methods = append(methods, "basicAuth")
 	}
 
-	if len(authInfo.ClientCertificate) != 0 || len(authInfo.ClientCertificateData) != 0 {
+	if len(authInfo.ClientCertificate) != 0 {
 		// Make sure cert data and file aren't both specified
```

Comment:
Seems like the code comments aren't valid any more :)

Reply:
please keep the same logic of ensuring that file and data can't both be specified